src/components/Submit/index.tsx

```diff
@@ -0,0 +1,49 @@
+import cn from 'classnames';
+import * as React from 'react';
+import { useRouteData, useSiteData } from 'react-static';
+
+import { Button, IButton } from 'src/components/Button';
+import { IInput, Input } from 'src/components/Input';
+import { useSubmitForm } from 'src/hooks/useSubmitForm';
+import { Container } from '../Container';
+
+export interface ISubmit {
+  button: IButton;
+  input: IInput;
+  formId: string;
+  portalId?: string;
+  className?: string;
+}
+
+export const Submit: React.FunctionComponent<ISubmit> = ({ button, className, input, formId }) => {
+  const { integrations } = useSiteData();
+  const { submit } = useRouteData(); // Do we still need this since formId is passed directly in now?
+
+  const [value, setValue] = React.useState('');
+  const [loading, response, submitForm] = useSubmitForm(integrations.hubspot, formId);
+
+  return (
+    <div className={cn(className, 'p-12 flex flex-wrap sm:flex-col sm:justify-center sm:items-between')}>
```

Comment:
Let's remove the padding from this component. If you need padding, the parent component can just pass it in as the classname 😁 

```suggestion
    <div className={cn(className, 'flex flex-wrap sm:flex-col sm:justify-center sm:items-between')}>
```

Reply:
Done